crates/verifier/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "sp1-verifier"
 description = "Verifier for SP1 Groth16 and Plonk proofs."
-readme = "../../README.md"
+readme = "README.md"
```

Comment:
on one hand, this crate is pretty independent, so it deserves its own readme.

on the other hand, i'm gonna write some docs in the mdbook, so i can just have this point to the workspace readme maybe?

Reply:
Doesn't matter